src/components/nodes-table/ExpandedNode.vue

```diff
@@ -53,6 +53,11 @@ export default {
     HomeAssistant,
     NodeDetails
   },
+  computed: {
+    showHass () {
+      return this.node.hassDevices && Object.keys(this.node.hassDevices).length > 0
```

Comment:
Isn't it true that you configure hassDevices with this tab ?
In other words: shouldn't you tie the visibility of the HA tab to the mqtt being enabled OR some toggle to enable HA MQTT discovery ?

Reply:
You configure the ones I have discovered, you canno create new. If discovery is disabled no one will be created here

Reply:
BTW settings have been completely refactored now. It's more clear and easy to use the gateway as only control panel